packages/cyberstorm/src/components/PackageSearch/PackageSearch.tsx

```diff
@@ -53,20 +55,29 @@ export function PackageSearch(props: Props) {
 
       <div className={styles.contentWrapper}>
         <div className={styles.sidebar}>
-          <SectionMenu
-            allSections={allSections}
-            selected={section}
-            setSelected={setSection}
-          />
+          <Suspense
+            fallback={range(1, 6).map((x) => (
```

Comment:
This "render x SkeletonBoxes" seems to be a recurring pattern. Could be nice to have something like `<SkeletonList items="6" className={styles.sideBarSkeletonBar} />` that would wrap the range-stuff?